test/test_other.py

```diff
@@ -214,7 +214,7 @@ def test_emcc_v(self):
       # from clang would be flushed to stderr first.
       self.assertContained('emcc (Emscripten gcc/clang-like replacement', proc.stderr)
       self.assertTrue(proc.stderr.startswith('emcc (Emscripten gcc/clang-like replacement'))
-      self.assertContained('clang version %s' % shared.EXPECTED_LLVM_VERSION, proc.stderr)
+      self.assertContained('clang version ', proc.stderr)
```

Comment:
Do we still have another test for the right version in the output? Specifically, will we get a test error that notifies us we need to bump the version number, or do we need to remember manually to do so?

Reply:
Yes, pretty much *all* tests will fail the version check in shared.py... but only in gitub CI and not in chromium CI.